dashboard/config/scripts_json/csd3-2023.script_json

```diff
@@ -20919,599 +20893,479 @@
       }
     },
     {
```

Comment:
The evidence levels have an issue with changing order, so this diff is really noisy. I did not edit any evidence levels.